media/js/base/mozilla-utils.js

```diff
@@ -43,21 +43,6 @@ if (typeof Mozilla === 'undefined') {
         return '/' + newLang + '/' + urlpath + location.search;
     };
 
-    // language switcher
-    Utils.initLangSwitcher = function() {
```

Comment:
There's still a piece of code calling this function. In the console I see:

```javascript
TypeError: utils.initLangSwitcher is not a function
```

Reply:
Can we also remove `Utils.switchPathLanguage` above? (plus the unit tests)

Reply:
I can't find the string `initLangSwitcher` in my code base.

✂️ `switchPathLanguage`